functionary/.vscode/launch.json

```diff
@@ -47,6 +47,17 @@
       "envFile": "${workspaceFolder}/.vscode/.debug.env",
       "django": true,
       "justMyCode": true
+    },
+    {
+      "name": "run_scheduler",
+      "type": "python",
+      "request": "launch",
+      "preLaunchTask": "scheduler_stop_container",
+      "program": "${workspaceFolder}/manage.py",
+      "args": ["run_scheduler"],
+      "envFile": "${workspaceFolder}/.vscode/.debug.env",
+      "django": true,
+      "justMyCode": false
```

Comment:
Did you mean to default this to false?

Reply:
Yes, this was on purpose.  The scheduler is pretty much all celery code, we just wrap it with a django management command to launch it.  So if this is set to True, there's not much to meaningful step through in the even of doing some debugging.